# Tank Drive
int8 left
int8 right

# All Wheel Speed
int8 wheel0
int8 wheel1
int8 wheel2
int8 wheel3
int8 wheel4
int8 wheel5

# Ride Height
int8 rh_front
int8 rh_back